src/components/TableCell/TableCell.tsx

```diff
@@ -0,0 +1,57 @@
+import { useContext } from 'react';
+import Tooltip from '@mui/material/Tooltip';
+import TemperatureUnits from 'constants/temperatureUnits';
+import SettingsContext from 'contexts/SettingsContext';
+import ForecastBody from 'types/forecastBody';
+import SettingsContextType from 'types/settingsContextType';
+
+interface TableCellProps {
+  weatherReport: ForecastBody[];
+}
+
+export default function TableCell({ weatherReport }: TableCellProps) {
```

Comment:
It should be either `TableRow` because the component renders multiple cells or it should render only one cell